### DISCUSSION TITLE
srsRAN gNB with srsUE: PBCH-MIB: CRC failed

### DISCUSSION THREAD

**Discussion by zenghuacheng** [srsran/srsRAN_Project#232] [General Help] [answered]
I am trying to connect srsUE and srsRAN gNB following this tutorial (https://docs.srsran.com/projects/project/en/latest/tutorials/source/srsUE/source/index.html). I do not have USRP B210. So I hope to use USRP X310 as the gNB and N310 as the  UE. I can run both gNB and srsUE successfully, but they do not connect with each other. I kept getting "PBCH-MIB: CRC failed" on UE side. The configuration files and UE/gNB log files are attached. I would appreciate it if you can provide any suggestion to troubleshoot the problem.

[gNB.yml](https://www.dropbox.com/scl/fi/ke8zu1nbde0qhkj30l80e/gnb_rf_x310_fdd_n3_10mhz.yml?rlkey=56bj63fxg3r2yuvjv8kghznpf&dl=0)
[gNB.log](https://www.dropbox.com/scl/fi/l2iqb8bi8wfpmsgkm7rrv/gnb.log?rlkey=4o45fw93iepus2w4yl1hr87h1&dl=0)
[ue.conf](https://www.dropbox.com/scl/fi/ydtkbemuuk9og6tg5wc9f/ue_rf.conf?rlkey=kqrltp239yonqwowkrgt7pgja&dl=0)
[ue.log](https://www.dropbox.com/scl/fi/7oav0mxxjn1aqx1nug1qi/ue.log?rlkey=qc9kt3ivrht4zmnb3hdgdsvlw&dl=0)

HZ

**Comment by pgawlowicz** [comment, marked as answer]
Please use the newest releases of both srsRAN_project (gnb) and srsRAN_4G (srsUE), and follow the updated tutorial (with new configs).
